scripts/test_resources/test_changelog.md

```diff
@@ -0,0 +1,3757 @@
+# Changelog for the Mapbox Navigation SDK for Android
```

Comment:
Could we prepare a more abstract (and shorter) test cases? I'd like to avoid duplicates in search queries and just save on pure size to maintain.